### ISSUE TITLE
Log parsing issues when using newtxmath

### ISSUE THREAD

**Issue by Filip-K** [SublimeText/LaTeXTools#1451]
The following minimal file
```
\documentclass{article}

\usepackage{newtxmath}

\author{a}
\title{b}

\begin{document}
\maketitle
\end{document}
```
gives me "Log parsing issues" with [this log](https://pastebin.com/RhFuvh0E). Any ideas? My file path doesn't have spaces or Unicode.

**Comment by r-stein** [comment]
You get a log parsing error from LaTeXTools? Unfortunately that happens with some log entries, because there is no standardized LaTeX log format and hence we just try to heuristically parse the file.

**Comment by Filip-K** [comment]
Yes, I get a LaTeXTools log parsing error. The document seems to be building just fine though.
